users/toomanydaves/game/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content"width=device-width,initial-scale=1">
+        <title>Sean's Game</title>
+        <script src="js/main.js"></script>
+        <link href="css/main.css" rel="stylesheet">
```

Comment:
Where are these links? I don't see them in the folder or are you planning to create those later?

Reply:
Yeah, I was planning on adding the files later as I start building the game (and the website). But probably better to just add some stubs so that there are no 404 error messages in the console. Thanks for the nudge in the right direction :-)